.circleci/unittest/linux/scripts/environment.yml

```diff
@@ -1,3 +1,4 @@
+# a comment for these trying times
```

Comment:
i don't have the hearth to say no to this comment :)

Reply:
It's an easter egg now!

😄